buildSrc/src/main/kotlin/dependencies/Dependencies.kt

```diff
@@ -21,6 +21,7 @@ object Dependencies {
         const val daggerAssistedInject = "0.3.2"
         const val expandableRecyclerView = "3.0.0-RC1"
         const val billing = "1.2"
+        const val apache = "3:3.0"
```

Comment:
I believe `3:` here should still be moved to the library name, as it's part of the library name. The version number is `3.0`. https://mvnrepository.com/artifact/org.apache.commons/commons-lang3

Reply:
Ah, sorry about that!